### PR TITLE
howtoread/index.md: explain scope of the handbook

### PR DESCRIPTION
**Pull request by bobertlo**
This is a follow-up to the issue #51

I would sincerely like input on this. I hope this is not too opinionated or in conflict with anyone else's opinions.

### CODE REVIEW

File: src/howtoread/index.md [hunk 1]
```diff
@@ -1,6 +1,24 @@
 # How To Read The Handbook
 
-## Examples
+This handbook is not an extensive guide on how to use and configure common Linux
+software. The purpose of this document is to explain how to install, configure,
+and maintain Void Linux systems, including the differences from other common
```
Comment (Vaelatern) [inline src/howtoread/index.md]: `Focusing on`, perhaps?

File: src/howtoread/index.md [hunk 2]
```diff
@@ -1,6 +1,24 @@
...
+Linux distributions.
+
+Those looking for tips and tricks on how to configure a Linux system in general
+should consult upstream software documentation. Additionally, the [Arch
+Wiki](https://wiki.archlinux.org/) provides a fairly comprehensive outline of
+common Linux software configuration, as well as the
+[google](https://www.google.com) search engine.
```
Comment (Vaelatern) [inline src/howtoread/index.md]: `and a variety of internet search engines exist for further assistance` maybe.

File: src/howtoread/index.md [hunk 3]
```diff
@@ -1,6 +1,24 @@
 # How To Read The Handbook
 
-## Examples
+This handbook is not an extensive guide on how to use and configure common Linux
+software. The purpose of this document is to explain how to install, configure,
+and maintain Void Linux systems, focusing on the differences from other common
```
Comment (nilium) [inline src/howtoread/index.md]:
```suggestion
and maintain Void Linux systems, and to explain differences between common Linux distributions and Void.
```

The wording of this seemed a little off.  Maybe explain is the wrong word too. Highlight? Cover? Something like that. I think what I'm trying to get at is that it's not that this guide focuses on the differences, just that there will be differences and this guide documents them.

Comment (bobertlo) [reply src/howtoread/index.md]: I did feel iffy about the wording here. I will come up with something better. I always appreciate your reviews and this is very helpful :)

**Comment by bobertlo** [comment]
@nilium I made those changes. I'm not sure this is the perfect wording, but much clearer.

If there are no other comments, I think this is ready to go.

**Comment by nilium** [comment]
I have nothing to add — seems fine to me.

**Pull request merged**